
Allow recode to be built with imap and mysql.  The conflict is due
to symbol conflicts between the recode libraries and anything else,
but these should not matter (right? right! everything is fine, really)
if the extensions are loaded as DSOs *and* the libraries are loaded
using the magic RTLD_DEEPBIND.

--- php-5.2.8/ext/recode/config9.m4.recode
+++ php-5.2.8/ext/recode/config9.m4
@@ -4,9 +4,6 @@ dnl
 
 dnl Check for extensions with which Recode can not work
 if test "$PHP_RECODE" != "no"; then
-  test "$PHP_IMAP"  != "no" && recode_conflict="$recode_conflict imap"
-  test "$PHP_MYSQL" != "no" && recode_conflict="$recode_conflict mysql"
-
   if test -n "$recode_conflict"; then
     AC_MSG_ERROR([recode extension can not be configured together with:$recode_conflict])
   fi
